### ISSUE TITLE
Any plan to publish/release it ?

### ISSUE THREAD

**Issue by noirbizarre** [baloise/rocket-chat-notifier#1]
Hi !

Thanks for this plugin ! Any plan to publish/release it ?

**Comment by culmat** [comment]
In terms of jenkins' integrated plugin manager - not really at that stage. But you can always [get the hpi from jitpack](https://github.com/baloise/rocket-chat-notifier/wiki/Get-it-now).
Does this help you?

**Comment by noirbizarre** [comment]
Yes, I had it installed. But, as I was using this before https://github.com/jenkinsci/slack-plugin
I'm wondering if it's planned to have a way to:
- choose which jobs notify, which one does not ?
- choose which phase is notified (job start, job end, success, failure...)
- publish in a given room depending on the job
- to customize the message sent

**Comment by culmat** [comment]
we use our rocket.chat server as a one to many messaging infrastructure to control extreem feedback devices for hundreds of jobs. So I always try to avoid manual configuration in individual jobs.
How do you imagine the features? 
As build steps / post build actions?
A central place where you can paste some groovy / js hooks?

**Comment by chkal** [comment]
Actually the [Instant Messaging Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Instant+Messaging+Plugin) provides most of the advanced features out of the box. Many of the IM plugins (like IRC, Jabber, Skype) are built on top of this base plugin.

But as far as I can tell this requires to configure the notifications per job. I agree with @culmat that this is cumbersome for large Jenkins setups.

**Comment by dhanish2k** [comment]
I've installed the plugin but can you please tell me how to configure it within  a Jenkins job?
